### ISSUE TITLE
Peer crawler port field type inconsistency

### ISSUE THREAD

**Issue by mDuo13** [XRPLF/rippled#5029]
## Issue Description
The ["Peer Crawler" special API method](https://xrpl.org/docs/references/http-websocket-apis/peer-port-methods/peer-crawler/) reports the `port` of peers as either an integer or a string depending on if the connection is inbound/outbound. This was probably an oversight in the original PR, and the field should be consistently the same type.

## Steps to Reproduce
Call the peer crawler on a server that has both incoming and outgoing peers and check the response.

## Expected Result
The `port` fields in the `overlay.active[]` array members should all be the same type, probably `Number`. Port numbers should always fit in any reasonable primitive that JSON numbers can be deserialized into.

## Solution
Always make `port` a Number type. Note, this is (kind of) an API change, but to a special peer-port-API method and not to the standard API, so API versioning concerns do not apply.

## Environment
`rippled` 1.2.1 through 2.1.x

## Related issues

- [Docs ticket noting the issue](https://github.com/XRPLF/xrpl-dev-portal/pull/2597)
- [Source code](https://github.com/XRPLF/rippled/blob/40b4adc9cc296a7e3c6e8c94b5a977a54c835613/src/ripple/overlay/impl/OverlayImpl.cpp)
- [Original Ticket (RIPD-866)](https://ripplelabs.atlassian.net/browse/RIPD-866)
- [Original PR](https://github.com/XRPLF/rippled/pull/1017)

**Comment by intelliot** [comment]
> Note, this is (kind of) an API change, but to a special peer-port-API method and not to the standard API, so API versioning concerns do not apply.

Agreed. Let's make sure it gets at least a mention in the release notes though.
